prover/setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
add a small comment on what this script does (that it sets up necessary data for the CPU fri prover locally).

Reply:
Added here [1f3dd01](https://github.com/matter-labs/zksync-era/pull/258/commits/1f3dd014e0db01124e80d1c2e4c4ad1d148a63d4)